src/blockchainTx/BlockchainTxService.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { BlockchainUserDto } from './data_models/BlockchainUserDto';
+import {NewIcoDto} from './data_models/NewIcoDto';
+
+import {ContractsService} from './web3/ContractsService';
+import {CentralizedBlockchainUserDto} from './data_models/CentralizedBlockchainUserDto';
+import {NewTokenDto} from './data_models/NewTokenDto';
+import {UpdateScreenerDto} from './data_models/UpdateScreenerDto';
+import {UserRemovalDto} from './data_models/UserRemovalDto';
+import {SwapRecoveryCentralizedDto} from './data_models/SwapRecoveryCentralizedDto';
+import {AddExtraRecoveryCentralizedDto} from './data_models/AddExtraRecoveryCentralizedDto';
+
+@Injectable()
+export class BlockchainTxService {
```

Comment:
Not sure why do we even need this service if all it does is proxying calls to `ContractsService`

Reply:
This might be a byproduct of abstracting the project. I see clearly what you mean. However, I would like to keep it as it is as I think that blockchaintxservice might need to query for more information later on from database and do some more synchronizing and returning. In this case I would just like contractsservice to deal with posting to blockchain and no other logic. Is that okay for now?